zeppelin-web/bower.json

```diff
@@ -41,15 +41,50 @@
     "ace-builds": {
       "main": [
         "src-noconflict/ace.js",
-        "src-noconflict/mode-scala.js",
-        "src-noconflict/mode-python.js",
-        "src-noconflict/mode-sql.js",
-        "src-noconflict/mode-markdown.js",
-        "src-noconflict/mode-sh.js",
-        "src-noconflict/mode-r.js",
+		"src-noconflict/mode-scala.js",
```

Comment:
The indention does not look aligned with the parent "main" above. Previous one was.

Also, adding such changes to PR that implements new functions makes code review task harder (and slower) - now everyone who reads this need to think, if this changes are part of the new feature or not.